### ISSUE TITLE
Fix / add feed for r-weekly

### ISSUE THREAD

**Issue by brshallo** [rweekly/rweekly.org#888]
Thanks for all the great work at r-weekly!

I think I may have submitted my feed incorrectly. I noticed at least one of my posts was retweeted by rweekly_live [here](https://rweekly.org/submit) though you can see the formatting is all messed-up. While I still show-up on the [feedlist](https://rweekly.org/feedlist) I am wondering if I might have been removed (due to potential error in my submission) as I noticed my [next post](https://www.bryanshalloway.com/2020/12/08/weighting-classification-outcomes/) was not mentioned by r-weekly or tweeted by rweekly_live.

*rss feed for my site (for rstats topics):* https://www.bryanshalloway.com/categories/rstats/index.xml 
*Description:* Solving problems using rstats and tidyverse tools.
*Twitter handle:* brshallo

Please let me know if I should resubmit or do anything to get this corrected?

**Comment by qinwf** [comment]
Thanks for the report, the feed is in the list. 

live.rweekly.org got the latest post link on 2021-03-31, but the post date is 2021-03-18, so it was not tweeted.

**Comment by brshallo** [comment]
Cool, thanks for the explanation. (I am assuming the format is OK then as well and the tweet I referenced was just an abnormality in how it was picked-up.)

It sounds like I should avoid doing that if I want it to be included on anything r-weekly related...?

(I frequently will publish a post, text it to a couple friends for feedback or tweet about it myself. Then I sometimes make a few edits based on notes I receive at which point I'll add the rstats category to the post -- which is the rss feed I provide to R-weekly and R-bloggers and determines when you receive it.) 

If I update the date but don't change the slug, would it be picked-up by r-weekly? Or is r-weekly looking at the date in the slug?

**Comment by qinwf** [comment]
Yes, if the date is updated to the recent day and the URL is different from before, and then it will be a new link for live.rweekly.org/

**Comment by brshallo** [comment]
Sorry, should have been more clear. If I update the date, but don't change the slug, would new posts get picked-up or do I need to also update the slug for it to be included? 

(There may be cases when I'd be hesitant to change the slug, so am just interested in which r-weekly is looking at to determine inclusion -- the slug or the date indicated under the title of the post).
